dist/style.css

```diff
@@ -5,12 +5,14 @@
 
 .view-card-container,
 .review-card-container,
+.update-card-container,
```

Comment:
Looks like a lot of CSS classes need these styles. Can you use the bootstrap container instead?